docs/application/dotnet/get-started/wearable/first-xaml-app.md

```diff
@@ -0,0 +1,318 @@
+2. In the command prompt, enter `sdb devices`.
+
+    ![Emulator detection](media/cs_first_building_sdb_prompt.png)
+
+    A list of the attached devices appears.
+
+If you face any issues during deployment, it is recommended to manually install the application using SDB:
+
+```bash
+$ sdb install <path-to-package>/org.tizen.example.TizenWearableXamlApp1-1.0.0.tpk
+```
+
+## Enhance Your Application
+
+Tizen Wearable XAML application adopts and integrates Xamarin.Forms XAML, which is mostly used to define the visual contents of a page and works together with a C# code-behind file.
+If you create your application using **Tizen Wearable Xaml App** template, you can update the UI and the business logic layer separately as they are not coupled and add different layout-xaml files for different configurations. 
+
+For more information about creating a XAML application, see [Part 1. Getting Started with XAML](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/xaml-basics/get-started-with-xaml).
+
+### Understand the Source Code
+
+The C\# code from your first application displays a label centered on the screen, containing the **Welcome to Xamarin.Forms!** text. This XAML application created from the template is set up and ready to be built and run by Visual Studio right after you create it, as described above.
+
+**TizenWearableXamlApp1** project has a pair of files: `App.xaml` is the XAML file and `App.xaml.cs` is the C# code-behind file associated with the XAML file. Both of them contribute to a class named `App` that derives from `Application`.
+
+To see the codde-behind file, `App.xaml.cs`, click the arrow icon next to `App.xaml`.
+
+The other two XAML files in `TizenWearableXamlApp1` project are associated to a class that derives from `ContentPage` to define the visual contents of an entire page :
+- `MainPage.xaml`: The XAML file
+- `MainPage.xaml.cs`: The C# code-behind file
+
+The following code snippet shows `MainPage.xaml` file:
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
+<c:CirclePage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:c="clr-namespace:Tizen.Wearable.CircularUI.Forms;assembly=Tizen.Wearable.CircularUI.Forms"
+             x:Class="TizenWearableXamlApp1.MainPage">
+  <c:CirclePage.Content>
+    <StackLayout>
+      <Label Text="Welcome to Xamarin.Forms!"
+          VerticalOptions="CenterAndExpand"
+          HorizontalOptions="CenterAndExpand" />
+    </StackLayout>
+  </c:CirclePage.Content>
+</c:CirclePage>
+```
+
+The first XML namespace declaration means that tags defined within the XAML file with no prefix refer to classes in `Tizen.Wearable.CircularUI.Forms`, for example `CirclePage`. The second namespace declaration defines a prefix of `x`. This is used for several elements and attributes that are intrinsic to XAML itself and which are supported by other implementations of XAML. 
```

Comment:
Question.
>The first XML namespace declaration means that tags defined within the XAML file with no prefix refer to classes in `Tizen.Wearable.CircularUI.Forms`, for example `CirclePage`. 


According to the description, the first XML namespace declaration is referring `Tizen.Wearable.CircularUI.Forms`.

But in the sample code, 
the 1st namespace is : xmlns="http://xamarin.com/schemas/2014/forms",
and the XML namespace that is referring `Tizen.Wearable.CircularUI.Forms` is the 3rd one, xmlns:c.

So shouldn't this sentence be changed to the description of third XML namespace declaration defines a prefix of `c`?